### PR TITLE
Change nativeLogLevel to Error and call setNativeLogLevel while bean is initialized

### PR DESCRIPTION
**Pull request by USAMAWIZARD**
https://github.com/ant-media/Ant-Media-Server/issues/5662

### CODE REVIEW

File: src/main/java/io/antmedia/settings/ServerSettings.java [hunk 1]
```diff
@@ -394,7 +394,7 @@ else if (ILicenceService.LICENCE_TYPE_OFFLINE.equals(licenseService.getLicenseTy
 				offlineLicense = true;
 			}
 		}
-
+		setNativeLogLevel(LOG_LEVEL_ERROR);
```
Comment (mekya) [inline src/main/java/io/antmedia/settings/ServerSettings.java]: It's supposed to call with nativeLogLevel parameter. 
```
setNativeLogLevel(nativeLogLevel);
```

File: src/test/java/io/antmedia/test/settings/ServerSettingsTest.java [hunk 2]
```diff
@@ -32,7 +32,7 @@ public void testNativeLogLevel() {
 		ServerSettings settings = new ServerSettings();
 		
 		assertEquals(Logging.Severity.LS_WARNING, settings.getWebRTCLogLevel());
-		assertEquals(ServerSettings.LOG_LEVEL_WARN, settings.getNativeLogLevel());
+		assertEquals(ServerSettings.LOG_LEVEL_ERROR, settings.getNativeLogLevel());
```
Comment (mekya) [inline src/test/java/io/antmedia/test/settings/ServerSettingsTest.java:36]: This test code is checking the default value. It's good to fix it.

On the other hand, there is no check if `setNativeLogLevel` is called in  `setApplicationContext` method which is part of the fix. So we may encounter same problem again and we may need to spend time again for our previous fixes. 

Remember the manifesto item - https://antmedia.io/about-us/
- Solve problems permanently. Don’t waste time addressing the same issue multiple times.

**Review by mekya — APPROVED** [review]
Hi @USAMAWIZARD,
There are good number of opportunities that you can make the product more stable and let everyone in the team have some more time to focus on more valuable things then fixing the old bugs. 

Please look at the comments and remember the manifesto item

-Solve problems permanently. Don’t waste time addressing the same issue multiple times.

File: src/test/java/io/antmedia/test/settings/ServerSettingsTest.java (hunk 2 above)
Comment (USAMAWIZARD) [reply src/test/java/io/antmedia/test/settings/ServerSettingsTest.java:36]: Got it , I will update the PR

Comment (mekya) [reply src/test/java/io/antmedia/test/settings/ServerSettingsTest.java:36]: I've added following lines for the test code

https://github.com/ant-media/Ant-Media-Server/pull/5772/files#diff-98b52309ed36b413f4fd32a8be8fcf05fcd787de4f3d5c6d1d6429f43b9ed571R94

File: src/main/java/io/antmedia/settings/ServerSettings.java [hunk 3]
```diff
@@ -394,7 +394,7 @@ else if (ILicenceService.LICENCE_TYPE_OFFLINE.equals(licenseService.getLicenseTy
 				offlineLicense = true;
 			}
 		}
-
+		setNativeLogLevel(nativeLogLevel);
```
Comment (mekya) [inline src/main/java/io/antmedia/settings/ServerSettings.java:397]: I've changed the LOG_LEVEL_ERROR with the value `nativeLogLevel`

**Review by mekya — APPROVED** [review]
I've added the test code and resolve the issue. 

Let's add two coffees amount for your donations at this time as we agreed in the previous meeting. 
So you may pay attention more for these things and you'll have the opportunity to 
donate more to society ;)

**Pull request merged**